content/sensu-go/5.16/api/handlers.md

```diff
@@ -27,27 +27,44 @@ The `/handlers` API endpoint provides HTTP GET access to [handler][1] data.
 The following example demonstrates a request to the `/handlers` API endpoint, resulting in a JSON array that contains [handler definitions][1].
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/handlers -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET
+http://127.0.0.1:8080/api/core/v2/namespaces/default/handlers
+-H "Authorization: Bearer $SENSU_TOKEN"
 [
   {
     "metadata": {
-      "name": "slack",
-      "namespace": "default",
-      "labels": null,
-      "annotations": null
+      "name": "influx-db",
+      "namespace": "default"
     },
-    "command": "sensu-slack-handler --channel '#monitoring'",
+    "type": "pipe",
+    "command": "sensu-influxdb-handler -d sensu",
+    "timeout": 0,
+    "handlers": null,
+    "filters": null,
     "env_vars": [
-      "SLACK_WEBHOOK_URL=https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX"
+      "INFLUXDB_ADDR=http://influxdb.default.svc.cluster.local:8086",
+      "INFLUXDB_USER=sensu",
+      "INFLUXDB_PASSWORD=password"
     ],
+    "runtime_assets": null
```

Comment:
I mentioned this down further, but this will need a runtime asset. Well, _technically_ not, but then the command would need to be different. But really, this should include a runtime asset IMO and should probably point to sensu/sensu-influxdb-handler